Sources/Misc/Signing+ResponseVerification.swift

```diff
@@ -61,17 +64,36 @@ extension HTTPResponse where Body == Data {
             return .notVerified
         }
 
-        guard let signature = HTTPResponse.value(forCaseInsensitiveHeaderField: HTTPClient.responseSignatureHeaderName,
-                                                 in: headers) else {
+        guard let signature = HTTPResponse.value(
+            forCaseInsensitiveHeaderField: HTTPClient.ResponseHeader.signature.rawValue,
+            in: headers
+        ) else {
             Logger.warn(Strings.signing.signature_was_requested_but_not_provided(request))
 
             return .failed
         }
 
-        if signing.verify(message: body,
-                          nonce: nonce,
-                          hasValidSignature: signature,
-                          with: publicKey) {
+        guard let requestTimeString = HTTPResponse.value(
+            forCaseInsensitiveHeaderField: HTTPClient.ResponseHeader.requestTime.rawValue,
+            in: headers
+        ), let requestTime = Int(requestTimeString) else {
+            Logger.warn(Strings.signing.request_time_missing_from_headers(request))
```

Comment:
Only log if signature is requested?

Reply:
This method starts with:
```swift
guard let nonce = request.nonce,
      let publicKey = publicKey,
      #available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.2, *) else {
    return .notVerified
}
```
So this won't be executed if there is no signature requested.